dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/com.ibm.websphere.appserver.jakartaee-9.0.feature

```diff
@@ -7,8 +7,17 @@ IBM-ShortName: jakartaee-9.0
 Subsystem-Version: 9.0.0
 Subsystem-Name: Jakarta EE Platform 9.0
 -features=\
-  com.ibm.websphere.appserver.concurrent-2.0,\
-  com.ibm.websphere.appserver.webProfile-9.0,\
-  com.ibm.websphere.appserver.transaction-2.0
-kind=noship
-edition=full
+ com.ibm.websphere.appserver.el-4.0, \
+ com.ibm.websphere.appserver.jsp-3.0, \
+ com.ibm.websphere.appserver.servlet-5.0, \
+ com.ibm.websphere.appserver.jsonp-2.0, \
+ com.ibm.websphere.appserver.jsonb-2.0, \
+ com.ibm.websphere.appserver.transaction-2.0, \
+ io.openliberty.jakarta.cdi-3.0, \
```

Comment:
adding these private api features here should not be required? They should be pulled in transitively by whichever public features depend on them?

Reply:
Moreover why the change in direction to not update webProfile-9.0 and have jakartaee-9.0 pull in webProfile-9.0?

Reply:
jakartaee-9.0 isn't core either.

Reply:
Hi Completely agree regarding changes to jakartaee feature - This was a temporary solution to release the beta content, and ensure some private API's were made available. - I expect webprofile-9.0 and other tweaks to the features will be ready by next beta - (actually this week)